contrib/gazebo/sawyer/push.py

```diff
@@ -0,0 +1,34 @@
+"""
+Push task for the sawyer robot
```

Comment:
Please also put this file (and all other environments) into contrib.gazebo.envs, so as the match the top-level package organization of the rest of rllab.